indico/htdocs/sass/modules/_category.scss

```diff
@@ -418,3 +418,160 @@ ul.category-resource-qtip {
         }
     }
 }
+
+.category-overview {
+    > .title {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 30px;
+
+        .text {
+            font-size: 1.2em;
+            font-style: italic;
+            margin-left: 5px;
+            margin-right: 5px;
+        }
+
+        .navigation a {
+            padding: 5px;
+        }
+    }
+
+    > table:not(.day) {
+        border: 1px solid #aaa;
+        border-collapse: separate;
+        border-spacing: 1px;
+
+        tr {
+            background-color: #eee;
+        }
+    }
+
+    table {
+        width: 100%;
+        border-collapse: collapse;
+
+        th {
+            background-color: #444;
```

Comment:
I didn't look into it since I understood we don't want to spend the time to restyle those pages completely now.

Those values where in the HTML, I just moved them to the CSS to make it easier to modernize the style at a later time.